string label
float32 confidence
uint16 pix_lt_x
uint16 pix_lt_y
uint16 pix_rb_x
uint16 pix_rb_y
float32 x
float32 y
float32 z

